ThumbnailSequence.cpp

```diff
@@ -1560,7 +1560,7 @@ ThumbnailSequence::CompositeItem::CompositeItem(
 bool
 ThumbnailSequence::CompositeItem::incompleteThumbnail() const
 {
-	return dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
+	return m_pThumb == 0 && dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
```

Comment:
It shouldn't be related to std::auto_ptr(), as it's merely deprecated not broken in C++11. I strongly suspect Qt and scantailor were built in a way that resulted in incompatible ABI between the two. Different versions of compilers or compiler flags that affect the ABI.
It should be possible to workaround this problem (introducing an AbstractThumbnail class with isIncomplete() method), but I don't want to apply a workaround to a problem I don't fully understand.